fmt/format.cc

```diff
@@ -171,9 +183,13 @@ int safe_strerror(
 
     // Fallback to strerror if strerror_r and strerror_s are not available.
     int fallback(internal::Null<>) {
+#if !defined(UNDER_CE)
       errno = 0;
       buffer_ = strerror(error_code_);
       return errno;
+#else
+	  return 0;
```

Comment:
Would it make sense to write some generic error message if strerror if not available?

Also the indentation is off.

Reply:
I don't know.
Which function in fmt sets `errno` in case of failure?
I can chack what happens in this kind of situations in CE6 and see how to handle in the best way.

Reply:
Thinking more of it, I think it's fine as is since the strerror message is normally used only as a part of error message.